pkg/controller/translator.mock.go

```diff
@@ -0,0 +1,95 @@
+// Automatically generated by MockGen. DO NOT EDIT!
+// Source: github.com/kubernetes-csi/external-provisioner/pkg/controller (interfaces: ProvisionerCSITranslator)
```

Comment:
There are a couple things bugging me about this. Primarily,
- I don't know how to update this mock if I change the interface.
- There's no import statement referencing MockGen, so I don't know where it came from (beyond a google search)
- I don't know how someone is supposed to use this mock. Its own godoc isn't helping. I guess I should read [this](https://godoc.org/github.com/golang/mock/gomock)?

Update: I read it and it answered most of my questions.

Reply:
ack. followed the style of other tests in this repo but agree it's not super clear. But also I learned it to write these tests and it didn't take long so I'm not too concerned about difficulty